collector/collector.go

```diff
@@ -150,6 +156,8 @@ func (c *graphiteCollector) processLine(line string) {
 	}
 
 	if (mappingPresent && mapping.Action == mapper.ActionTypeDrop) || (!mappingPresent && c.strictMatch) {
+		c.logger.Warn("Dropped line", "line", line)
```

Comment:
I think it is best to remove this on make it debug level. Some people running this on a high throughput scenario would see lots of allocation here due to string copying and would also pollute logs a lot.

```suggestion
```